ManatiCR/manatime
# 75: Make day names translatable.

**Steps to test**
- Create a timesheet node.
- Go to /timesheet and check if the day names look good.

[issue](https://github.com/ManatiCR/manatime/issues/75)
